application_store_api/namespace_fund/routes.py

```diff
@@ -0,0 +1,96 @@
+from application_store_api.namespace_fund.data_store.store import APPLICATIONS
+from application_store_api.namespace_fund.fund_ns import (
+    application_model_inbound,
+)
+from application_store_api.namespace_fund.fund_ns import fund_ns
+from flask_restx import reqparse
+from flask_restx import Resource
+
+
+# GET/DELETE all funds
+@fund_ns.route("/all_funds")
+class Fund(Resource):
```

Comment:
Here is a cool python feature - if one has a class or function definition then you can place a "docstring" beneath it like so:

```python
def is_adam_is_cool(day_of_the_week : Str) -> bool:
  """
  A function which returns true if Adam is cool on that day of the week, false if not.
  
      Args:
          day_of_the_week (Str): The day of the week given as a string.
  
      Returns:
          bool : Returns True if Adam is cool on that day, False if not.
  """
  if day_of_the_week[:3] in ["mon","tue","wed","thu","fri","sat","sun"]:
     return True
```

This allows autodocs and IDEs to describe the code.

I personally believe that if one is providing a description of a function then you can write it as a docstring and get lots of free goodies (IDE integration is a nice one) without any extra work. It also means that proper code comments are reserved for deeper, lower level code descriptions.

Feel free to disagree with me.

Reply:
i wasnt aware of the autodocs feature to be fair - very good to know. Thanks Jack :)